### ISSUE TITLE
make check error

### ISSUE THREAD

**Issue by pfe1223** [vicentebolea/scheduling-simulator#2]
When I run `make check` I get the following error:

```bash
stdin_reader.cc: In member function ‘virtual std::vector<std::__cxx11::basic_string<char> > scheduler_simulator::StdinReader::next()’:
stdin_reader.cc:29:8: error: ignoring return value of ‘char* fgets(char*, int, FILE*)’, declared with attribute warn_unused_result [-Werror=unused-result]
   fgets(line, SCHEDULER_SIMULATOR_INPUT_LINE_LENGTH, stdin);
   ~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

**Comment by vicentebolea** [comment]
@pfe1223 Thanks for your interest.

Find this being resolved at #3. Note that there is no CI here so perhaps you might want to locally check the branch at that Pull Request

**Comment by vicentebolea** [comment]
@pfe1223 nevermind I just added a simple CI check
